packages/scss/src/components/_action-icon.scss

```diff
@@ -27,10 +27,9 @@
 
 .actionIcon {
 	&.mod-invert {
-		color: #CCC;
+		color: white;
```

Comment:
You might want to make it a variable. If I'm correct, this is used to display actionIcon on dark bg. Not sure white will always be our go-to pick. What do you think ?

Reply:
Yup, I hesitated...
Let's add it on theming.